wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -336,6 +337,15 @@ public final class ZMUserSession: NSObject {
             context: syncContext)
     }()
 
+    public lazy var selfMLSClientID: MLSClientID? = {
+        guard let selfUserClient  else {
+            return nil
+        }
+       return  MLSClientID(userClient: selfUserClient)
+    }()
```

Comment:
Please remove this and just call `MLSClientID(userClient:)` directly wherever it is needed.

Reply:
done